generate-event-config.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+: ${srcdir=$(dirname "$0")}
+
+: ${AWK=awk}
+: ${CP=cp}
+: ${MV=mv}
+: ${SED=sed}
+
+set -eu
+
+private_header="$srcdir/evconfig-private.h.in"
+input_file="$srcdir/config.h"
+output_file="$1"
+
+tempfile=$(mktemp config.h.XXXXXXXX)
+trap 'rm -f $output_file $tempfile' EXIT INT TERM
+
+private_constants=$($SED -E -ne 's/#[[:space:]]*undef[[:space:]]+//p' < "$private_header")
+
+$CP "$input_file" "$tempfile"
+for private_constant in $private_constants; do
+	$SED -E -i "" \
```

Comment:
while `sed -i "" ..` can work on non-linux it will not work under linux, since it requires the following `sed -i""` (no space after `-i`)

Reply:
Ugh. I forgot about that -_-.